gui/src/index.html

```diff
@@ -1781,8 +1781,25 @@ <h5 class="modal-title">Settings</h5>
                     <option value="alpha">alpha</option>
                   </select>
                 </div>
+                <div class="input-group input-group-sm mb-1">
+                  <label class="input-group-text w-50"
+                    >Auto-start TNC/rigctld</label
+                  >
+                  <label class="input-group-text w-50">
+                    <div class="form-check form-switch form-check-inline">
+                      <input
+                        class="form-check-input"
+                        type="checkbox"
+                        id="AutoStartSwitch"
```

Comment:
Also adding a Hamlib indicator here

Reply:
The text/label clearly states rigctld/tnc.  Is this not sufficient?

Reply:
I can't see the full code actually because I'm at work. If you're saying it's ok, then we can use this naming. Just wanted at least talking about this for being sure